llvm/lib/Object/ELFObjectFile.cpp

```diff
@@ -517,6 +519,73 @@ StringRef ELFObjectFileBase::getAMDGPUCPUName() const {
   }
 }
 
+StringRef ELFObjectFileBase::getNVPTXCPUName() const {
```

Comment:
getNVPTXGPUName() or getNVPTXArchName()?

Reply:
I wasn't sure which to call it. AMDGPU calls everything `processor` so I copied the format for the name here.